src/components/LandingPages/ClientLanding.tsx

```diff
@@ -1,27 +1,143 @@
-import React, { Component } from 'react';
+import React, { Component, CSSProperties } from 'react';
 import { Helmet } from 'react-helmet';
-import { Link, RouteComponentProps, withRouter } from 'react-router-dom';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
 
-import AppSVG from '../../static/images/calendar.svg';
-import FileSVG from '../../static/images/file.svg';
-import SignDoc from '../../static/images/sign-document.png';
+import getServerURL from '../../serverOverride';
+import AtWorkPic from '../../static/images/atwork-rafiki.png';
+import DocumentsPic from '../../static/images/documents-rafiki.png';
 import BaseCard, { CardImageLoc, CardSize } from '../BaseComponents/BaseCard';
 
 interface Props extends RouteComponentProps {
   name: String
+  username: String
 }
 
-class ClientLanding extends Component<Props, {}, {}> {
+interface State {
+  activities: Array<any>,
+  isLoading: Boolean
+}
+
+const activitiesCardStyles: CSSProperties = {
```

Comment:
I think this can be a larger conversation with front end on whether we want to have styling in our typescript files or have it in our scss files. I would probably prefer to have them in scss files to stay consistent so that if people are looking for styling/css, they know where to go.